src/data/global.json

```diff
@@ -280,7 +327,7 @@
                 }
             ],
             "copyright": {
-                "title": "© 2020-2022 - The authors of the Corona-Warn-App open-source project",
+                "title": "© 2020-2021 - The authors of the Corona-Warn-App open-source project",
```

Comment:
The copyright year has been reverted. It should go back to 2020-2022.

Reply:
Thanks @MikeMcC399! Fixed it in the latest commit.